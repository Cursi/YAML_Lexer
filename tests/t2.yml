---
# maps
mapone: {key1: value, key2: value2}
maptwo: {key1: {a: b, other: 22}, key2: value}