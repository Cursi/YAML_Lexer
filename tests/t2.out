| document: 
| | map value
| | | key: [mapone]
| | | map value
| | | | key: [key1]
| | | | string value: [value]
| | | | key: [key2]
| | | | string value: [value2]
| | | key: [maptwo]
| | | map value
| | | | key: [key1]
| | | | map value:
| | | | | key: [a]
| | | | | string value: [b]
| | | | | key: [other]
| | | | | numeric value: [22]
| | | | key: [key2]
| | | | string value: [value]
